docs/theory-of-operation.md

```diff
@@ -0,0 +1,208 @@
+  output to COM2.
+  >NOTE: In above example serial connection is redirect to telnet. First, it is
+  opened on 13541 port (COM1). After enabling redirect console to COM2, output
+  is no longer available. New telnet connection is opened on 13542 port (COM2)
+  and console output is available there.
+
+
+- **Enable/Disable CPU boost**
+
+  This feature lets user `Enable/Disable` CPU performance boost. To verify if
+  it works, memory test can be done (in boot menu choose `payload[memtest]`).
+
+  >NOTE: Notice how memory transfer speed changes depending on CPU boost
+  enable/disable.
+
+  Watch [video](https://asciinema.org/a/241571) showing how to enable/disable
+  CPU performance boost and verify it.
+
+
+- **Enable/Disable UARTC/UARTD**
+
+	This feature lets user enable/disable superIO UARTx on GPIO header.  
+
+	Watch [Video](https://asciinema.org/a/241576) showing how to enable/disable
+	UARTC/UARTD.
+
+
+- **Enable/Disable mPCIe2 clk**
+
+  If mPCIe2 clk is enabled then GPP3 PCIe clock (which is attached to apu2
+  mPCIe2 slot) is always on. It is used when extension card is attached to
+  mPCIe2 slot. If no extension card is attached it is advised to set to
+  `Disable`.
+
+  Watch [Video](https://asciinema.org/a/239813) showing how to enable/disable
+  mPCIe2 clk.
+
+
+- **Enable/Disable EHCI0 controller**
+
+	This feature lets user enable/disable EHCI0 controller.
+
+	Watch [Video](https://asciinema.org/a/239816) showing how to enable/disable
+	EHCI0 controller.
+
+
+- **Restore default sortbootorder settings**
+
+  If user wants to bring back default settings in sortbootorder, it can be
+  restored by typing `r` in payload menu. It will reset enable/disable features
+  and boot order to defaults.
+
+  Watch [video](https://asciinema.org/a/240935) showing how to restore settings
+  to its default values.
+
+
+- **Press `F10` button to enter boot menu and boot menu 6s timeout**
+
+	During boot performance, press `F10` button to enter boot menu.
+
+  >NOTE: After power on user has 6 seconds to enter boot menu. If no button is
+  pressed then automatically boot is performed.
+
+  Watch [Video](https://asciinema.org/a/240500) showing how to enter boot menu
+  via `F10` button and boot menu 6s timeout feature.
+
+
+- **Screen refresh during memory test performance**
+
+  To perform memory test choose `Payload [memtest]` option in boot menu.
+
+  During memory test user can refresh screen if the output in terminal is not
+  available. It happens when serial connection is opened during test
+  performance. Screen refresh can be done by typing `l` or `L`.
+
+  Watch [Video](https://asciinema.org/a/241564) showing how screen refresh
+  during memory test works.
+
+- **TPM SHA1 and SHA256 banks enable/disable**
+
+  PC Engines apu2 supports TPM module. This feature lets user choose which PCR
+  banks are active.
+
+  Watch [video](https://asciinema.org/a/241633) showing how to enable/disable
```

Comment:
@piotr-kleins the video does not clearly show the PCR banks. The size of the window is small and it cannot be seen when the banks are disabled. Please print PCRs with `less` command or so, so that anyone can scroll down slowly and see which banks are available.